packages/core/src/components/tag/tag.css

```diff
@@ -4,29 +4,36 @@
 .hds-tag {
   --tag-background: var(--color-black-10);
   --tag-color: var(--color-black-90);
+  --tag-font-size-large: var(--fontsize-body-m);
   --tag-font-size: var(--fontsize-body-s);
-  --tag-spacing: var(--spacing-2-xs);
+  --tag-height-large: 48px;
   --tag-height: 32px;
+  --tag-padding-large: 10px 20px;
+  --tag-padding: 4px 12px;
+  --tag-spacing-large: var(--spacing-s);
+  --tag-spacing: var(--spacing-xs);
 
   align-items: center;
   background-color: var(--tag-background);
-  border-radius:    20px;
+  border-radius: calc(var(--tag-height) / 2);
+  box-sizing: border-box;
   color: var(--tag-color);
   display: inline-flex;
-  flex-direction:   row;
+  flex-direction: row;
   font-size: var(--tag-font-size);
-  line-height: var(--tag-height);
+  gap: var(--spacing-2-xs);
+  justify-content: center;
+  line-height: 24px;
```

Comment:
Figmassa käytetään tarkkoja pikseleitä. Pitäisikö koodissa kuitenkin käyttää tokeneita? (Pohdinnassa vaihtaa Figmaan myös tokenien mukaiset prosentuaalit käyttöön)

Reply:
changed to use line-height -vars